packages/components/Tabs/src/Tabs.windows.tsx

```diff
@@ -0,0 +1,177 @@
+/** @jsx withSlots */
+import * as React from 'react';
+import { View } from 'react-native';
+import { Text } from '@fluentui-react-native/text';
+import { tabsName, TabsType, TabsProps, TabsState, TabsSlotProps, TabsRenderData, TabsContextData } from './Tabs.types';
+import { compose, IUseComposeStyling } from '@uifabricshared/foundation-compose';
+import { ISlots, withSlots } from '@uifabricshared/foundation-composable';
+import { settings } from './Tabs.settings';
+import { mergeSettings } from '@uifabricshared/foundation-settings';
+import { filterViewProps } from '@fluentui-react-native/adapters';
+import { foregroundColorTokens, textTokens, backgroundColorTokens } from '@fluentui-react-native/tokens';
+import { useSelectedKey, useAsPressable } from '@fluentui-react-native/interactive-hooks';
+
+export const TabsContext = React.createContext<TabsContextData>({
+  selectedKey: null,
+  onTabsClick: (/* key: string */) => {
+    return;
+  },
+  getTabId: (/* key:string, index: number*/) => {
+    return null
+  },
+  updateSelectedTabsItemRef: (/* ref: React.RefObject<any>*/) => {
+    return;
+  },
+  tabsItemKeys: [],
+  views: null,
+  focusZoneRef: null,
+});
+
+export const Tabs = compose<TabsType>({
+  displayName: tabsName,
+
+  usePrepareProps: (userProps: TabsProps, useStyling: IUseComposeStyling<TabsType>) => {
+    const focusZoneRef = React.useRef(null);
+    const defaultComponentRef = React.useRef(null);
+    const {
+      label,
+      accessibilityLabel = userProps.label,
+      selectedKey,
+      headersOnly,
+      defaultSelectedKey,
+      getTabId,
+      componentRef = defaultComponentRef,
+      isCircularNavigation,
+      ...rest
+    } = userProps;
+
+    // This hook updates the Selected TabsItem and calls the customer's onTabsClick function. This gets called after a TabsItem is pressed.
+    const data = useSelectedKey(selectedKey || defaultSelectedKey || null, userProps.onTabsClick);
+
+    const onChangeTabId = React.useCallback((key: string, index: number) => {
```

Comment:
I'm not sure the name of this function is intuitive. The function is to get the tab id, right? The function name sounds like your changing the tabID. I'd change it to `findTabId` or something similar

Reply:
Yeah, making the change to ```findTabId``` on Tabs and Tabs.windows.